tests/testthat/test-init.R

```diff
@@ -23,7 +23,7 @@ testthat::test_that("init data accepts teal_data object", {
   )
 })
 
-testthat::test_that("init data throws an error with input other than TealData, teal_data and ddl", {
+testthat::test_that("init data throws an error with input other than TealData, teal_data and list(ui, server)", {
```

Comment:
This passes:
```
init(
  data = teal_data_module(ui = function(id) div(), server = function(id) NULL),
  modules = example_module()
)
```

but this fails:
```
init(
  data = list(ui = function(id) div(), server = function(id) NULL),
  modules = example_module()
)


Error in to_relational_data.list(data = data) :    Assertion on 'data' failed: May only contain the following types: {dataset,data.frame,MultiAssayExperiment,TealDataset,TealDatasetConnector}, but element 1 has type 'function'.
```

Reply:
Sorry, I will not address it. `to_relational_data` will be removed and thus `teal::init` will accept only `teal_data` and `teal_data_module`. Currently you're theoretically able to pass `list(ui = function(), server = function())` and  also `list(ui = iris, server = MAE)`. This state is just temporary and will be solved soon

Reply:
ok then